package.json

```diff
@@ -13,9 +13,9 @@
     "pretest": "npm run build",
     "test": "cross-env NODE_ENV=test nyc mocha dist/tests/**/index.spec.js --timeout 80000 --exit",
     "clean": "rm -rf dist",
-    "coverage": "nyc report --reporter=text-lcov|coveralls",
+    "coverage": "npm test && nyc report --reporter=text-lcov | coveralls",
```

Comment:
please can you change the coverage script to the above

Reply:
@obasajujoshua31  what is the above?

Reply:
> @obasajujoshua31 what is the above?

Done in latest commit